src/hooks/useBuildPipelines.ts

```diff
@@ -22,22 +22,12 @@ export const useBuildPipelines = (
             [PipelineRunLabel.APPLICATION]: applicationName,
             [PipelineRunLabel.PIPELINE_TYPE]: PipelineRunType.BUILD,
           },
-          ...(includeComponents &&
-            componentNames?.length > 0 && {
-              matchExpressions: [
-                {
-                  key: PipelineRunLabel.COMPONENT,
-                  operator: 'In',
-                  values: componentNames,
-                },
-              ],
-            }),
```

Comment:
we should add the creation timestamp filter here also. This also has the potential of fetching the deleted pipeline runs

Reply:
updated all where requested